lints/lint_issuer_multiple_rdn.go

```diff
@@ -0,0 +1,45 @@
+package lints
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
+}
+
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
```

Comment:
This is from AWS Certlint, and it looks like they check this on every certificate. But I can't find it in an RFC or the BR, so unfortunately I don't have the text to support it